apps/docs-snippets/src/guide/provider/querying-the-chain.test.ts

```diff
@@ -17,40 +19,33 @@ describe('querying the chain', () => {
       [100, assetIdA],
     ]);
 
-    // get single coin
-    const coin = await wallet.getCoins(baseAssetId);
-    // [{ amount: bn(42), assetId: baseAssetId }]
+    // fetches up to 100 coins from baseAssetId
+    const { coins, pageInfo } = await provider.getCoins(wallet.address, baseAssetId);
+    // [
+    //   { amount: bn(42), assetId: baseAssetId },
+    //   ...
+    // ]
 
-    // get all coins
-    const coins = await wallet.getCoins();
+    // fetches up to 100 coins from all assets
+    await provider.getCoins(wallet.address);
```

Comment:
Given `coins` and `pageInfo` are being initialized in [line23](https://github.com/FuelLabs/fuels-ts/pull/2408/files#diff-2652c5c77082242be32a0b6b929773f36a3c9a5f31598b0bbff64ee16632e28cR23), how are these results being tested?

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2408#discussion_r1647586635